### PR TITLE
Removed redundant includes

### PR DESCRIPTION
**Pull request by WBigaj**

### CODE REVIEW

File: shm/cargo.cpp [hunk 1]
```diff
@@ -1,7 +1,6 @@
 #include "cargo.hpp"
 
 #include <iostream>
```
Comment (Marcinati) [inline shm/cargo.cpp]: we can delete this also?

File: shm/cargo.cpp [hunk 2]
```diff
@@ -1,7 +1,6 @@
...
-#include <string>
```
Comment (Marcinati) [inline shm/cargo.cpp:4]: good

File: shm/map.hpp [hunk 3]
```diff
@@ -1,7 +1,5 @@
 #pragma once
 
-#include <vector>
-
```
Comment (Marcinati) [inline shm/map.hpp:4]: ok

File: shm/player.cpp [hunk 4]
```diff
@@ -1,7 +1,6 @@
 #include "player.hpp"
 
 #include <algorithm>
-#include <memory>
```
Comment (Marcinati) [inline shm/player.cpp:4]: ok

File: shm/ship.cpp [hunk 5]
```diff
@@ -1,7 +1,6 @@
 #include "ship.hpp"
 
 #include <iostream>
-#include <string>
```
Comment (Marcinati) [inline shm/ship.cpp]: correct, and iostream can delete also

File: shm/cargo.cpp (hunk 1 above)
Comment (WBigaj) [reply shm/cargo.cpp]: Done

**Review by Marcinati — APPROVED** [review]

**Pull request merged**